### DISCUSSION TITLE
Migrating an existing payment method

### DISCUSSION THREAD

**Discussion by j2is** [swellstores/community#155] [Help] [answered]
I'm trying to migrate a customer's card details that was captured inside of Stripe but isn't reflected in Swell.
Using the [guide](https://developers.swell.is/guides/migrate-to-swell) I see the instruction is:

```
POST /accounts/{id}/cards {
  "id": "60424e018c28b57c6c5b302b",
  "token": "card_1J0SQEG77uxvnJDE56oKJMdS",
  "stripe_token": "card_1J0SQEG77uxvnJDE56oKJMdS",
  "stripe_customer": "cus_JdjuUTWMalp6JD"
}
```

I can't seem to find card_ in the transaction details, is it pm_ now?
When I post it asks for more details such as parent_id, do I need to include more than what is referenced in the guide then? Such as fingerprint, date_created etc.?

Thanks so much

**Comment by sags95** [comment, marked as answer]
@j2is We do support migrating `pm_` tokens as well, we will correct the docs to reflect these tokens. In your case the request would look like this:

```
POST /accounts/{id}/cards {
  "token": "<pm_***>",
  "stripe_token": "<pm_***>",
  "stripe_customer": "<cus_***>"
}
```

The rest of the fields should be imported automatically after passing in these details
